src/Installer/core-sdk-tasks/GetWorkloadSetFeatureBand.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class GetWorkloadSetFeatureBand : Task
+    {
+        [Required]
+        public string WorkloadSetVersion { get; set; }
+
+        [Output]
+        public string WorkloadSetFeatureBand { get; set; }
+
+        public override bool Execute()
+        {
+            WorkloadSetFeatureBand = Microsoft.DotNet.Workloads.Workload.WorkloadSetVersion.GetFeatureBand(WorkloadSetVersion).ToString();
```

Comment:
nit: Can we capture some of this in a `using` statement?

Reply:
Done